app/controllers/idv/doc_auth_controller.rb

```diff
@@ -62,7 +62,7 @@ def check_for_outage
       if VendorStatus.new.any_ial2_vendor_outage?
         session[:vendor_outage_redirect] = current_step
         session[:vendor_outage_redirect_from_idv] = true
-        redirect_to vendor_outage_url
+        redirect_to VendorStatus.new.vendor_outage_url_localized
```

Comment:
we already have some helpers for this:
https://github.com/18F/identity-idp/blob/a1b05b6349b4e8a7988b1ebc5e38df312c701672/app/controllers/application_controller.rb#L92-L94
https://github.com/18F/identity-idp/blob/a1b05b6349b4e8a7988b1ebc5e38df312c701672/app/helpers/locale_helper.rb#L2-L5


So I think another option would be:

```suggestion
        redirect_to vendor_outage_url(default_url_options)
```

Now that I write that out, we're already in a controller so I would expect that `default_url_options` would be automagically included already...? Any ideas why this didn't pick up that param?

Reply:
ok, so I just tested it and it actually was picked up for the vendor outage page - my confusion was due to a bug where entry to idv reverts to English. I'll close this PR.

Thanks!